lib/internal/url.js

```diff
@@ -546,6 +547,7 @@ class URL {
     input = `${input}`;
     this[context] = new URLContext();
     this.#onParseComplete = FunctionPrototypeBind(this.#onParseComplete, this);
+    this.#onSearchUpdate = FunctionPrototypeBind(this.#onSearchUpdate, this);
```

Comment:
Binding an arrow function has no useful effect.
```suggestion
```

Reply:
Nice catch. Removed it. Thanks!